<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLogin</class>
 <widget class="QDialog" name="frmLogin">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Watermarking System - Login </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QPushButton" name="btnRegister">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLGN">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPass">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Your ID number</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 255, 240);
color: rgb(255, 255, 255);
border-color: rgb(244, 255, 217);</string>
   </property>
   <property name="text">
    <string> LOG IN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../login/log.PNG</normaloff>../login/log.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>85</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>New user? Click here:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>WhatsThisCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Watch a video online to see how this software works</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 231, 255);</string>
   </property>
   <property name="text">
    <string> HELP</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../help.ico</normaloff>../../help.ico</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="cboxShow">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
